store/actions/PatientActions.js

```diff
@@ -1,11 +1,34 @@
 import {
     HYDRATE_PATIENT
 } from './types'
+import { dummy_Patient_Data } from '../../research/FakePatientData';
 
-
-export const hydratePatientData = (data) => { // expects an array as argument
-    const patientData = { id: data } // TODO: hydrate all data
+//Will be imporing a patientId, which is a bar code number called 'data'
+export const hydratePatientData = (data) => { 
+    const patientData = {};
+    
+    
     return (dispatch) => {
+
+        for (let i = 0; i < dummy_Patient_Data.length; i++){
+            console.log("Kevin dummy: data " + data + "  jsonID " + dummy_Patient_Data[i].patient_Id); 
+            if (dummy_Patient_Data[i].patient_Id == data) {
+                patientData.id = dummy_Patient_Data[i].patient_Id;
```

Comment:
What you can do here is use the `.find()` method, rather than creating the loop with the conditional inside

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find